authenticator7

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
```

Comment:
This should probably use the [stdlib csv reader](https://docs.python.org/3.7/library/csv.html?highlight=csv#module-csv) instead of pandas.